From: David Milburn <dmilburn@redhat.com>
Date: Wed, 12 Aug 2009 14:04:56 -0500
Subject: [ata] ahci: add device ID for 82801JI sata controller
Message-id: 20090812190456.GA4559@dhcp-210.hsv.redhat.com
O-Subject: [RHEL5.5 PATCH] ahci: add device ID for 82801JI sata controller
Bugzilla: 506200
RH-Acked-by: Jeff Garzik <jgarzik@redhat.com>
RH-Acked-by: Pete Zaitcev <zaitcev@redhat.com>
RH-Acked-by: Stefan Assmann <sassmann@redhat.com>

Add device ID for Intel 82801JI SATA AHCI contoller.

Customer has verified a -128.1.14.el5 test kernel
built with this patch.

upstream commit b2dde6afe5d29212d521e69492ebc299db235001

Please review and ACK.

Thanks,
David

 drivers/ata/ahci.c |    1 +
 1 files changed, 1 insertions(+), 0 deletions(-)

diff --git a/drivers/ata/ahci.c b/drivers/ata/ahci.c
index 069eb5e..495094a 100644
--- a/drivers/ata/ahci.c
+++ b/drivers/ata/ahci.c
@@ -478,6 +478,7 @@ static const struct pci_device_id ahci_pci_tbl[] = {
 	{ PCI_VDEVICE(INTEL, 0x502a), board_ahci }, /* Tolapai */
 	{ PCI_VDEVICE(INTEL, 0x502b), board_ahci }, /* Tolapai */
 	{ PCI_VDEVICE(INTEL, 0x3a05), board_ahci }, /* ICH10 */
+	{ PCI_VDEVICE(INTEL, 0x3a22), board_ahci }, /* ICH10 */
 	{ PCI_VDEVICE(INTEL, 0x3a25), board_ahci }, /* ICH10 */
 	{ PCI_VDEVICE(INTEL, 0x3b22), board_ahci }, /* PCH AHCI */
 	{ PCI_VDEVICE(INTEL, 0x3b23), board_ahci }, /* PCH AHCI */
